var/spack/repos/builtin/packages/miopentensile/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Miopentensile(CMakePackage):
```

Comment:
Hi. I was wondering why you do not have this inherit from ROCmPackage lib/spack/spack/build_systems/rocm.py, and then add related dependencies in that common place? 

So this line would become  `class Miopentensile(CMakePackage, ROCmPackage):` and some version information would go to that file?

I have not been involved in spack for a while. If there is something deficient/missing/incorrect in that approach for your application then that is ok, no problem.